nncf/parameters.py

```diff
@@ -119,6 +119,23 @@ class CompressionFormat(StrEnum):
     FQ_LORA = "fake_quantize_with_lora"
 
 
+@api(canonical_alias="nncf.StripFormat")
```

Comment:
This should appear in the nncf API docs after the merge, please check everything is shown properly after the merge